doc/tutorials/tutorial_understanding_building_benchmark.rst

```diff
@@ -0,0 +1,312 @@
+
+where
+    - \\(y\\) is a vector with \\(m\\) elements, typically some observations
+    - \\(X\\) is a matrix with \\(m \\times n\\) elements, typically the regressors
+    - :math:`\beta` is a vector with \\(n\\) elements that we obtain by regressing \\(y\\) on \\(X\\).
+    - \\(\\|x\\|\\) is the euclidean norm of \\(x\\).
+     
+
+Let us also suppose that we are working with a specific optimization algorithm that aims at solving the ridge regression problem: `the gradient descent (GD) algorithm <https://en.wikipedia.org/wiki/Gradient_descent>`_. GD is a simple algorithm that moves iteratively along the steepest descent direction. Given an estimate :math:`\beta`, GD iterates the following step:
+
+.. math:: 
+
+    \beta \leftarrow \beta - \eta \nabla_{f}(\beta)
+
+where :math:`\eta` is the learning rate or stepsize, and controls how much the unknown is moved alongside the descent direction. Vector :math:`\nabla_{f}(\beta)` is the gradient of the cost function defined above, evaluated at \\(\\beta\\).
+
+Even though GD is a simple algorithm, when applied to our ridge regression problem, we should be mindful of the two hyperparameters: the regularization parameter \\(\\lambda\\), and the gradient stepsize \\( \\eta\\). It is very common in machine learning that we are facing a situation where we want to vary these parameters, and compare the behavior of our algorithm in each scenario. Benchopt we can create a benchmark for this problem, here is a screenshot of what we will get at the end of this tutorial.
+
+.. image:: screenshot_mytemplate_ridgeloss.png
+    :width: 800
+
+We can now dive into the framework and see how leverage it to easily create a benchmark!
+
+..
+    - It is also fairly common to have to compare several different optimization algorithms to minimize the same cost function. For anyone who has experience in this, performing fair comparisons between various algorithms, which are typically implemented by other people, is quite challenging to do properly. For instance, our implementation of GD may be incorrect. If we trust the algorithm to return a loss function which is computed inside the GD code, it may potentially be artificially small. For instance, when implementing GD, we might mistakingly return as the loss :math:`\frac{1}{2} {\|y - X\beta\|_2}^2 + \frac{1}{2}\lambda{\|\beta\|_2}^2` where the regularization has been wrongly halved. Then this implementation of GD would appear much better, but in fact it is being unfairly compared to competitors and the results reported would be inaccurate.
+
+
+Installing Benchopt
+-------------------
+
+Let's start by installing the benchopt package. It is a python package, so we are assuming that you are starting this tutorial with a functioning python installation, preferably version >=3.10. It is recommanded to run all this tutorial in a dedicated environment (such as a `virtual environment <https://docs.python.org/fr/3/library/venv.html>`_ or a conda environment)
```

Comment:
I'd also remove the mention of virtual environments and conda environments (orthgonal to the tuto IMO). YMMV

Reply:
removed the python version bit.

If possible I would like to keep the mention of virtual environment (at least until more reviews suggest we remove it), it is a good practice anyways and will avoid that people have issues with the tutorial because of a cluttered (base) environment.